pkg/helpers/config/userdata.yaml

```diff
@@ -20,8 +20,8 @@ write_files:
       fi
       echo "${USERDATA_END}" >> /var/log/userdata-output
 runcmd:
-  - sudo service docker start 2>1 > /dev/null || echo "docker not started by systemctl"
-  - /run-container.sh
+  - sudo systemctl start docker 2>1 > /dev/null || echo "docker not started by systemctl"
+  - HTTP_PROXY=${HTTP_PROXY} HTTPS_PROXY=${HTTPS_PROXY} /run-container.sh
```

Comment:
fyi this enables `sudo docker pull` in the body of the userdata to work correctly when `HTTP_PROXY` and `HTTPS_PROXY` are defined (was missing before), however it still will not work if the customer provides an additional trust bundle.